script.module.inputstreamhelper/lib/inputstreamhelper.py

```diff
@@ -131,16 +116,21 @@ def _kodi_version(cls):
         version = xbmc.getInfoLabel('System.BuildVersion')
         return version.split(' ')[0]
 
+    @classmethod
+    def _legacy(cls):
```

Comment:
`return LooseVersion('18.0') > cls._kodi_version()` would be enough (minor)

Reply:
@enen92 Yeah, that's a brainfart on my part. I will make sure to fix that for the next version. Thanks! :-)